core/src/test/scala/unit/kafka/server/ReplicationQuotasTest.scala

```diff
@@ -230,13 +218,31 @@ class ReplicationQuotasTest extends ZooKeeperTestHarness {
 
   private def waitForOffsetsToMatch(offset: Int, partitionId: Int, brokerId: Int): Boolean = {
     waitUntilTrue(() => {
-      offset == brokerFor(brokerId).getLogManager.getLog(TopicAndPartition(topic, partitionId)).map(_.logEndOffset).getOrElse(0)
+      offset == brokerFor(brokerId).getLogManager.getLog(TopicAndPartition(topic, partitionId))
+        .map(_.logEndOffset).getOrElse(0)
     }, s"Offsets did not match for partition $partitionId on broker $brokerId", 60000)
   }
 
   private def property(key: String, value: String) = {
-    new Properties() { put(key, value) }
+    val props = new Properties()
+    props.put(key, value)
+    props
   }
 
   private def brokerFor(id: Int): KafkaServer = brokers.filter(_.config.brokerId == id)(0)
-}
+
+  def createBrokers(brokerIds: Inclusive): Unit = {
```

Comment:
Why are you specifying `brokerIds` as `Inclusive` instead of just a `Seq[Int]`? The latter works for ranges and manually specified broker ids.